packages/flutter/lib/src/rendering/custom_paint.dart

```diff
@@ -234,6 +238,8 @@ abstract class CustomPainter extends Listenable {
   /// repaints should be avoided as much as possible, a [RepaintBoundary] or
   /// [RenderRepaintBoundary] (or other render object with
   /// [RenderObject.isRepaintBoundary] set to true) might be helpful.
+  ///
+  /// `oldDelegate` is never null.
```

Comment:
Avoid starting sentences with a lower case letter, e.g. "The `oldDelegate` argument will never be null."

Reply:
Oops. We should add a linter.